designsystem/src/commonMain/kotlin/org/hisp/dhis/mobileui/designsystem/component/Text.kt

```diff
@@ -47,5 +55,78 @@ internal fun InputShellLabelText(
     textColor: Color,
     modifier: Modifier = Modifier
 ) {
-    Text(text, modifier = modifier, color = textColor, style = MaterialTheme.typography.titleSmall, textAlign = TextAlign.Start)
+    Text(
+        text,
+        modifier = modifier,
+        color = textColor,
+        style = MaterialTheme.typography.titleSmall,
+        textAlign = TextAlign.Start
+    )
+}
+
+class PrefixTransformation(val prefix: String) : VisualTransformation {
```

Comment:
shouldn't these class be in specific files?

Reply:
also if we don't want to expose them, we can create a package inside components called internal and place them there with the internal annotation?

Reply:
Good idea! I'll refactor it